services/logging-operator/4.2.3/defaults/cm.yaml

```diff
@@ -25,7 +24,7 @@ data:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: logging-operator-logging-4.1.0-d2iq-defaults
+  name: logging-operator-logging-4.2.3-d2iq-defaults
```

Comment:
i looked at the [fluent-bit version](https://github.com/kube-logging/logging-operator/blob/e183d85815b3111da679704101a3c5b7f6d28b02/pkg/sdk/logging/api/v1beta1/logging_types.go#L129) they ship by default now, its `2.1.4`

one of these bumps we should update that tag in here and in the `fluent-bit` app